raster/r.support/main.c

```diff
@@ -252,10 +284,21 @@ int main(int argc, char *argv[])
 	Rast_free_cats(&cats);
     }
 
+    if (bandref_opt->answer) {
+        if (Rast_legal_bandref(bandref_opt->answer) < 0)
+            G_fatal_error(_("Provided band reference is not valid. "
+                            "See documentation for valid examples"));
+
+        Rast_write_bandref(infile, bandref_opt->answer);
+    }
+
+    if (bandref_rm_flag->answer)
+        G_remove_misc("cell_misc", "bandref", infile);
 
     if (title_opt->answer || history_opt->answer || units_opt->answer
 	|| vdatum_opt->answer || datasrc1_opt->answer || datasrc2_opt->answer
-	|| datadesc_opt->answer || map_opt->answer)
+	|| datadesc_opt->answer || map_opt->answer
+    || bandref_opt->answer || bandref_rm_flag->answer)
```

Comment:
This block has mixed tab/space indentation.

Reply:
Most of C code base does not confirm to GRASS indentation rules. I made a proposal to re-indent whole code for 8.0, but so far only Python part has materialized. I could run grass_indent.sh on r.support, but that would change all files significantly thus cluttering the band reference part with formatting noise.

Reply:
I absolutely agree it is high time to batch run grass_indent.sh, with #1391 in place at least the whitespace changes of such operation may be "hidden".

Meanwhile, the main problem with this line (although minor) is not with the mixed use of space/tab, but the added line should have 8 spaces to line up with the tab on previous line.

Reply:
I replaced with a tab for consistency. Let the grass_indent.sh do the dirty job.